infra/aws/terraform/registry-k8s-io-prod/locals.tf

```
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+locals {
+  registry-k8s-io-prod-name       = "kops-infra-ci"
+  registry-k8s-io-prod-ci-index   = index(data.aws_organizations_organization.current.accounts.*.name, local.registry-k8s-io-prod-name)
```

Naming convention is `snake_case`

```suggestion
  registry_k8s_io_prod_ci_index   = index(data.aws_organizations_organization.current.accounts[*].name, local.registry-k8s-io-prod-name)
```

---

Done.